src/macho.rs

```diff
@@ -358,6 +537,126 @@ pub struct DyldCacheImageInfo<E: Endian> {
     pub pad: U32<E>,
 }
 
+/// Corresponds to struct dyld_cache_slide_pointer5 from dyld_cache_format.h.
+#[derive(Clone, Copy)]
+pub union DyldCacheSlidePointer5 {
```

Comment:
Can we avoid a union here? It seems like that would be possible if we stop using bitfields.